docs/kratos/passwordless/05_apple-passkeys.mdx

```diff
@@ -0,0 +1,45 @@
+---
+id: apple-passkeys
+title: Passwordless sign-in and sign-up with Apple Passkeys
+sidebar_label: Apple Passkeys
+---
+
+# Apple Passkeys
```

Comment:
Passkeys are supported and pushed by Apple, Google and Microsoft, the 3 most relevant platform vendors, and thus are about to become ubiquitous. We should make that clear!

For SEO purposes, we want "Apple Passkeys" on the page as a phrase, too, though.